packages/frontend/.env.example

```diff
@@ -1,3 +1,6 @@
 REACT_APP_INFURA_PROJECT_ID=
 REACT_APP_WALLETCONNECT_PROJECT_ID=
-REACT_APP_FATHOM_SITE_ID=
\ No newline at end of file
+REACT_APP_FATHOM_SITE_ID=
+
+export REACT_APP_FATHOM_SITE_ID=""
```

Comment:
Not sure export is needed here. The frontend picks up envs with dotenv (have a look at the `check-env` script)